spring-cloud-kubernetes-config/src/main/java/org/springframework/cloud/kubernetes/config/ConfigMapPropertySource.java

```diff
@@ -55,12 +57,13 @@
 	private static final String PREFIX = "configmap";
 
 	public ConfigMapPropertySource(KubernetesClient client, String name) {
-		this(client, name, null, (Environment) null);
+		this(client, name, null, null, (Environment) null);
 	}
 
 	public ConfigMapPropertySource(KubernetesClient client, String name, String namespace,
-			String[] profiles) {
-		this(client, name, namespace, createEnvironmentWithActiveProfiles(profiles));
+			RetryPolicy retryPolicy, String[] profiles) {
```

Comment:
I would prefer we not change the public constructor and instead create a new one (mark these two as deprecated)

Reply:
Perfect!